src/radical/pilot/agent/agent_0.py

```diff
@@ -4,13 +4,10 @@
 
 
 import os
-import sys
 import copy
 import stat
 import time
 import pprint
-import subprocess          as sp
-import multiprocessing     as mp
```

Comment:
deleted libs are in use below (L414, 427-428, 431)

Reply:
this is related to the comment below about `_start_services`, thus could be skipped if that method will be updated accordingly